README.md

```diff
@@ -46,57 +50,93 @@ be used for the schemaless are below:
 
 ### Running schemaless scripts
 
-Once CSV's from all of the data sources above have been downloaded, you must produce
-a `schemalesss` and `uuid_map` file. Below is an example of running these commands once on
-`02-27-2020` and then running it again to grab diffs on `03-04-2020`
+If you want to use the latest available data to create a schemaless file and
+have an internet connection, just run
+
+```sh
+pipenv shell
+
+python3 -m schemaless.create_schemaless \
+  --parcel_data_file=data/assessor/2020-02-18-parcels.csv.xz \
```

Comment:
Does this mean they still need to explicitly download the parcels file? If soo maybe we should make that clear above?

Also is the reason we need to do this because we need to pass it into subsequent calls? If so maybe we just file a github issue for Mike or someone else to tackle as a possible 'future refactor'

Reply:
Yes, I had mentally flagged this as a TODO. It's not exactly a `Source` so I didn't implement downloading it automatically. I'll be adding this when I also add support for automatically downloading the previous schemaless and UUID csvs for generating diffs.

Reply:
Created #169

Reply:
Ok sounds good, sounds like it's on our radar to do this